### PR TITLE
Setup & track notebook

### PR DESCRIPTION
**Pull request by falexwolf**
This PR makes some small adjustments so that the setup works for Alex.

### CODE REVIEW

**Comment by falexwolf** [comment]
Trying to reproduce the setup, I ran into a few issues and fixed them

1. No Jupyter notebook was installed in the docker -> I didn't feel it should be part of requirements, so I added them in both Dockerfile and Dockerfile-dev
2. No AWS profile was available, so, I commented it out for now -> it'd be great if there was docs on how to set it up!
3. lamindb was still 0.42.0 -> bumped to 0.43.0
4. The notebook command errored -> added `--allow-root`

In this process, I had to rebuild the lamin docker container twice to account for the notebook and the bump in lamindb, using
```
docker-compose up -d --no-deps --build lamin
```

I also deleted the postgres database once because it didn't store the config file (need to think through how to do this)
```
rm -r postgres-data
``` 

After I perform these steps (see changes in commits so far), I can setup an instance with local storage and interactively track the notebook:
image

File: Dockerfile-dev [hunk 1]
```diff
@@ -17,4 +17,5 @@ RUN sudo ./aws/install
 USER jovyan
 
 ADD requirements.txt /tmp/requirements.txt
-RUN pip install -r /tmp/requirements.txt
\ No newline at end of file
+RUN pip install -r /tmp/requirements.txt
+RUN pip install notebook
```
Comment (lawrlee) [inline Dockerfile-dev:21]: This shouldn't be necessary as it's already installed in the `jupyter/scipy-notebook` image. Were you unable to launch the notebook?

Comment (lawrlee) [reply Dockerfile-dev:21]: https://github.com/jupyter/docker-stacks/blob/main/base-notebook/Dockerfile#L38-L48

Comment (falexwolf) [reply Dockerfile-dev:21]: Ahhh, I knew it must have been somewhere

But yes, it kept giving me an error that the notebook command was installed when just following the steps in your readme

Manually adding it resolved it

**Pull request closed without merging**